server-common/src/main/java/com/datastrato/gravitino/server/auth/KerberosAuthenticator.java

```diff
@@ -52,6 +52,10 @@ public class KerberosAuthenticator implements Authenticator {
   public void initialize(Config config) throws RuntimeException {
     try {
       String principal = config.get(KerberosConfig.PRINCIPAL);
+      if (!principal.startsWith("HTTP/")) {
+        throw new IllegalArgumentException("Principal must starts with `HTTP/`");
```

Comment:
You should add the restrictions to the doc.

Reply:
Fixed.